src/main/java/team/silvertown/masil/common/util/DateTimeConverter.java

```diff
@@ -0,0 +1,24 @@
+package team.silvertown.masil.common.util;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.time.OffsetTime;
+import java.time.ZoneOffset;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class DateTimeConverter {
+
+    private static final ZoneOffset KrOffset = ZoneOffset.of("+09:00");
```

Comment:
이렇게 쓰면 좀 예뻐보이는 것 같아요!

```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class DateTimeConverter {

    private static final ZoneOffset KR_OFFSET = ZoneOffset.of("+09:00");
```